schemas.py

```diff
@@ -0,0 +1,37 @@
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
```

Comment:
Consider making the `bio` field optional if it is not always required. You can do this by setting it as `Optional[str] = None`.